### ISSUE TITLE
picEdit in asp.net c#

### ISSUE THREAD

**Issue by Fliflomo123** [andyvr/picEdit#22]
Is it possible to use PicEdit in asp.net? Because I am not able to get the image i have just edited

**Comment by simonpapworth6742** [comment]
Grap the second file, the first is the orginal.

I edited picedit to blank the fist one in my copy


Sent from Samsung Mobile

<div>-------- Original message --------</div><div>From: Fliflomo123 <notifications@github.com> </div><div>Date:25/03/2015  09:46  (GMT+00:00) </div><div>To: andyvr/picEdit <picEdit@noreply.github.com> </div><div>Subject: [picEdit] picEdit in asp.net c# (#22) </div><div>
</div>
Is it possible to use PicEdit in asp.net? Because I am not able to get the image i have just edited

---
Reply to this email directly or view it on GitHub:
https://github.com/andyvr/picEdit/issues/22

**Comment by Fliflomo123** [comment]
Yeah but I am not able to get one image. How can I save it in asp.net to the server?

**Comment by KokoRyuKai** [comment]
Using MVC 5 and if you click on the file option, pic photo and don't edit, and upload it's great, can pass other form data no problem, couldn't be happier.

But drag n drop, edited images and webcam sourced images are not passing the file at all, it's like the input type=file is empty... is there an onsubmit callback needed to populate the input with the correct/editted filename and path?
